RELEASE_PROCESS.md

```diff
@@ -0,0 +1,176 @@
+# Podman Releases
+
+## Overview
+
+Podman (and podman-remote) versioning is mostly based on semantic-versioning
+standards.
+Significant versions
+are tagged, including *release candidates* (`rc`).
+All relevant **major** releases have their own branches.  The **latest**
+development efforts occur on the *master* branch.  Branches with a
+*rhel* suffix are use for long-term support of downstream RHEL releases.
+
+## Release workflow expectations
+
+* You have push access to the [upstream podman repository][https://github.com/containers/podman.git]
+* You understand all basic `git` operations and concepts, like creating commits,
+  local vs. remote branches, rebasing, and conflict resolution.
+* You have access to your public and private *GPG* keys.
+* You have reliable internet access (i.e. not the public WiFi link at McDonalds)
+* Other podman maintainers are online/available for assistance if needed.
+* For a **major** release, you have 4-8 hours of time available, most of which will
+  be dedicatd to writing release notes.
```

Comment:
typo

Reply:
Got it, thanks.  I'm not going to push yet, since there are other updates I need to make.